### PR TITLE
Adding v4 documentation

### PR DESCRIPTION
**Pull request by KetanShrikhande**

### CODE REVIEW

**Comment by sarraloew** [comment]
Hi Ketan, the docs should be in the locate folder, with the version in the filesname like this: v4.user-locations.markdown. Also please add v1 and v4 to the page title. Thanks!

**Comment by KetanShrikhande** [comment]
Hello! Made the changes per the review comments above.

File: src/api-reference/locate/v1.userLocations.markdown [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: User Locations
+title: User Locations V1 (Deprecating soon)
```
Comment (charlieconcur) [inline src/api-reference/locate/v1.userLocations.markdown]: We shouldn't use 'deprecating soon' in the title. Use the official `includes` if you are going to deprecate AND we can create a new `includes` for this purpose if needed. See https://github.com/concur/developer.concur.com/blob/preview/src/_includes/deprecation-alert.html

**Review by charlieconcur — CHANGES_REQUESTED** [review]

File: src/api-reference/locate/v1.userLocations.markdown (hunk 1 above)
Comment (charlieconcur) [reply src/api-reference/locate/v1.userLocations.markdown]: @sarraloew Maybe you'd like to chime in here?

Comment (sarraloew) [reply src/api-reference/locate/v1.userLocations.markdown]: I've asked for more clarification around whether the v1 API was ever used- we may be able to just remove that documentation without doing the full deprecation process.

Comment (KetanShrikhande) [reply src/api-reference/locate/v1.userLocations.markdown]: Thanks! No problems I will wait for the direction forward.

**Comment by sarraloew** [comment]
I emailed the PM again, if I don't get a response soon we can publish this PR and handle the v1 deprecation separately.

File: src/api-reference/locate/v1.userLocations.markdown (hunk 1 above)
Comment (sarraloew) [reply src/api-reference/locate/v1.userLocations.markdown]: One client is still using v1, so for now, remove the Deprecating soon text, and I'll connect with the PM on the official deprecation process.

**Comment by sarraloew** [comment]
@KetanShrikhande Please remove the (Deprecating Soon) from v1, then we can merge this PR. Thanks!

**Review by charlieconcur — APPROVED** [review]

File: src/api-reference/locate/v1.userLocations.markdown (hunk 1 above)
Comment (charlieconcur) [reply src/api-reference/locate/v1.userLocations.markdown]: I removed it for @KetanShrikhande.

**Comment by charlieconcur** [comment]
:shipit:

**Comment by charlieconcur** [comment]
@sarraloew I removed the `(Deprecating Soon)` on behalf of @KetanShrikhande. This one is ready to be merged.

**Pull request merged**